R/utils-format.R

```diff
@@ -27,3 +27,7 @@ indent_print <- function(x) {
   mark <- if (identical(getOption("OutDec"), ",")) "." else ","
   formatC(x, big.mark = mark, ...)
 }
+
+dbplyr_use_colour <- function() {
+  getOption("dbplyr_use_colour", FALSE) && cli::num_ansi_colors() > 1L
+}
```

Comment:
I'd consider also adding

```R
style_kw <- function(x) {
  if (dbplyr_use_colour()) {
    cli::col_blue(x)
  } else {
    x
  }
}
```

And you don't need to check `num_ansi_colors()` since cli already does that for you.